apps/desktop-wallet/src/features/autoUpdate/AutoUpdateToastBox.tsx

```diff
@@ -12,7 +12,7 @@ import { openInWebBrowser } from '@/utils/misc'
 
 type UpdateStatus = 'download-available' | 'downloading' | 'download-finished' | 'download-failed'
 
-const AutoUpdateToastBox = () => {
+const AutoUpdateToastBox = memo(() => {
```

Comment:
Can't wait for the react compiler :)

Reply:
Same. I actually already integrated the eslint package of react compiler. We have quite some things to fix :) A PR will come after the release.